app/src/keymap.c

```diff
@@ -230,7 +231,16 @@ zmk_keymap_get_layer_binding_at_idx(zmk_keymap_layer_id_t layer_id, uint8_t bind
 
     ASSERT_LAYER_VAL(layer_id, NULL)
 
-    return &zmk_keymap[layer_id][binding_idx];
+    uint32_t *pos_map;
+    int ret;
+
+    ret = zmk_physical_layouts_get_selected_to_stock_position_map(&pos_map);
```

Comment:
```suggestion
    int ret = zmk_physical_layouts_get_selected_to_stock_position_map(&pos_map);
```
Can simplify this as is done later in the file